go.mod

```diff
@@ -10,7 +10,7 @@ require (
 	github.com/DataDog/zstd v1.5.2
 	github.com/NYTimes/gziphandler v1.1.1
 	github.com/antithesishq/antithesis-sdk-go v0.3.8
-	github.com/ava-labs/coreth v0.13.9-rc.2-encapsulate-signer
```

Comment:
Is there a corresponding PR to coreth for this change? Wondering what this diff is for

Reply:
quite minimal. I can't really create a final version until this one is merged since I'd need to tag it.. but these are the expected changes : https://github.com/ava-labs/coreth/pull/710